### PR TITLE
Visual Changes are Here! w00t!

### PR DESCRIPTION
**Pull request by Spedwards**
Initial commit includes global visuals: http://support.screeps.com/hc/en-us/community/posts/115000493525-PTR-Changelog-2017-02-22

### CODE REVIEW

File: visuals.js [hunk 1]
```diff
@@ -66,41 +66,23 @@ module.exports = class Visuals {
...
 	
-	static drawRoomInfo(room) {
-		const vis = new RoomVisual(room.name);
-		let x;
-		let y = 0;
-		// Room Name, centered middle
-		vis.text(`Room: ${vis.roomName}`, 24.5, ++y);
-		// Display bars: RCL, GCL, CPU, Bucket, Tick #
+	static drawGlobal() {
+		const vis = new RoomVisual();
```
Comment (karlthepagan) [inline visuals.js]: requires PTR

Comment (Spedwards) [inline visuals.js]: @karlthepagan It's time! :D

**Comment by karlthepagan** [comment]
I'll get this into canary and probably dev soon.

**Comment by pandabear41** [comment]
I was having an odd issue with this. Some visual stuff is showing up and others are not. Adding {font:0.7} to all text that don't have a font property fixes it.

**Comment by Spedwards** [comment]
If anyone can provide screenshots or even more information on the problem @pandabear41 is having, I'd really appreciate it. I unfortunately don't have the CPU to test this.

**Comment by pandabear41** [comment]
It seems to work now. I did see that yesterday afternoon they did push a small patch, since my client asked to be restarted after I came home from work.

**Pull request merged**